### DISCUSSION TITLE
I am a little confused with the Windows version

### DISCUSSION THREAD

**Discussion by capoei** [falkTX/Carla#1532] [Q&A]
can the host be included in the audion chains?
if not, what is it actualy usefull for?

**Comment by falkTX** [comment]
Confused by what exactly? By what a modular host is useful for?
I am the one confused by your question..

**Comment by capoei** [comment]
I load the audio-gain plugin for example. there is an output device, but I can't route anything into it.
I am talking about the standalone

**Comment by falkTX** [comment]
Converted this as a discussion because it doesnt make sense as ticket.

Anyway, you need to be a LOT more clear, because I just dont know what is your question at all.
you dont understand how to drag&drop connections? load plugins? what..?

**Comment by capoei** [comment]
Sorry, I am new to Windows audio.
I was asking myself how I could use it like in linux.
e.g.: I load a sepctrum analyser VST into Carla on Windows. how can I then, for example, conect the outputs of Chrome into the analyser?

**Reply by falkTX** [reply]
You can't. Windows does not provide stuff like that.

Maximum is applications that support ASIO through JACK-Router, but that is not very reliable of a setup.

**Comment by capoei** [comment]
I see. I suspected that. thanks
